tests/integration/files/conf/cloud.providers.d/profitbricks.conf

```diff
@@ -1,6 +1,6 @@
 profitbricks-config:
-  username: 'foo'
-  password: 'bar'
+  username: ''
```

Comment:
This was added because of the details in here: https://github.com/saltstack/salt/pull/46265  If we aren't seeing this issue anymore we can remove this. If you are not seeing that issue with this change I am guessing that we arent installing the profitbrick library or its been updated and fixed?

Reply:
I'm glad you pointed this out because it was confusing to me.  I'll verify that the issue you referenced won't come back